mace/cli/fine_tuning_select.py

```diff
@@ -136,12 +159,13 @@ def parse_args() -> argparse.Namespace:
         default=1.0,
     )
     parser.add_argument("--seed", help="random seed", type=int, default=42)
-    return parser.parse_args()
+    parser.add_argument("--atomic_numbers", help="atomic numbers to keep for filtering", nargs="+", type=int, required=False)
+    return parser
 
 
 def calculate_descriptors(atoms: List[ase.Atoms], calc: MACECalculator) -> None:
     logging.info("Calculating descriptors")
-    for mol in atoms:
+    for mol in tqdm(atoms,total=len(atoms), desc="Calculate descriptors"):
```

Comment:
doesn't tqdm default to `len(iterable)` ?

Reply:
you are right I just like to be explicit